docs/tenancy/1.x/hooks-migrations.md

```diff
@@ -11,8 +11,58 @@ tags:
     - seeds
 ---
 
-Install using composer:
+## Installation
+`Hooks-migration` is a package that is a bit more complex than most Tenancy packages. This will take care of everything after a Database is created for your multi database setup.
 
+
+### Hooks Migration
 ```bash
 composer require tenancy/hooks-migration
 ```
+
+#### Configuring Migrations
+Once you've installed the package, you will have to configure how Tenant should be migrated. You can this by listening to the `\Tenancy\Hooks\Migration\Events\ConfigureMigrations` event. It will provide you with some simple functionality in order to make your Migrations work like a charm:
```

Comment:
```suggestion
Once you've installed the package, you will have to configure how Tenant should be migrated. You can do this by listening to the `\Tenancy\Hooks\Migration\Events\ConfigureMigrations` event. It will provide you with some simple functionality in order to make your Migrations work like a charm:
```